src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import core.basesyntax.ball.Ball;
+import core.basesyntax.ball.Color;
+import java.util.Random;
+
+public class Lottery {
+    public String getRandomBall() {
+        int indexColor = new Random().nextInt(Color.values().length);
+        Color color = Color.values()[indexColor];
+        int number = new Random().nextInt(100);
```

Comment:
All magic numbers, strings, and symbols in your code should be constants.
[this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)